packages/services/index.js

```diff
@@ -210,10 +213,29 @@ const startRelayer = () =>
     resolve(startServer)
   })
 
+const startBridge = () =>
+  new Promise(resolve => {
+    console.log('Starting bridge server...')
+    const cwd = path.resolve(__dirname, '../../infra/bridge')
+    const startServer = spawn(`node`, ['src/app.js'], {
+      cwd,
+      stdio: 'inherit',
+      env: {
+        ...process.env,
+        DATABASE_URL: 'postgres://origin:origin@localhost/origin',
+        LOG_LEVEL: process.env.LOG_LEVEL || 'NONE'
```

Comment:
Shouldn't the `EMAIL_ATTESTATION_AUTOPASS` env be set here? Also should we set some default values for `ATTESTATION_ACCOUNT` and `ATTESTATION_SIGNING_KEY`? Attestations may not work locally without setting these env variables.

Reply:
For now I was thinking keeping this bridge setup lightweight and mostly for identity purposes.
For verification purposes, it seems easier to user bridge.dev.originprotocol.com since it is already fully configured for all attestation services.